commit 5018ca7d9ce64b8cd11e39a69575e7d2dec84554 (refs/changes/68/2068/3)
Author: Jared Morrow <jm@joyent.com>
Date:   2017-07-21T17:51:59-06:00 (2 years, 3 months ago)
    
    MANTA-2900 mlocate should not need to setupMantaBuckets
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/bin/mlocate b/bin/mlocate
index d592789..993e949 100755
--- a/bin/mlocate
+++ b/bin/mlocate
@@ -81,6 +81,7 @@ function createMorayClient(opts, cb) {
 
     var log = LOG.child({component: 'moray'}, true);
     opts.log = log;
+    opts.readOnly = true;
 
     var client = new libmanta.createMorayClient(opts);
 
